management-commands/config.md

```diff
@@ -89,16 +108,22 @@ MoonlightBot requires the following permissions to successfully execute this sub
 
 ## detect-assignment
 
-<TODO: describe>
+This command allows you to set up a detection system for a role that is assigned to a user by another bot or a
+moderator. It will change the role to a temporary role and remove it after a certain duration.
+
+{% hint style="warning" %}
+This setting requires that MonnlightBot has access to the audit logs to detect the assignment of roles by other bots or
```

Comment:
> MonnlightBot

Typo.

Reply:
@Darkfinst12 waiting for you to address this then I'll approve and merge.